build/common.ps1

```diff
@@ -189,6 +189,13 @@ Function Install-NuGet {
     & $NuGetExe locals all -list -verbosity detailed
 }
 
+Function Add-ApexFeed {
+    if (Test-Path $NuGetExe) {    
+        # Add Apex VSTS feed
+        & $NuGetExe sources add -Name ApexFeed -Source $env:VstsPackageFeedUrl -UserName $env:VstsPackageFeedUsername -Password $env:VstsPackageFeedPassword
```

Comment:
which config file will this get added to? the one in our repo or the one is %applocaldata% ?